Code/GraphMol/MolInterchange/details.h

```diff
@@ -7,6 +7,8 @@
 //  which is included in the file license.txt, found at the root
 //  of the RDKit source tree.
 //
+#include <GraphMol/Atom.h>
```

Comment:
These two lines should go inside the include guards

Reply:
Done